.cloud_build/dart_pad.yaml

```
@@ -1,12 +1,7 @@
 steps:
   - name: gcr.io/$PROJECT_ID/flutter:stable
```

I think this is going to need `main` still for some of the current fixes we're relying on. I'm not sure if `beta` would work or not, I haven't tried it.

```suggestion
  - name: gcr.io/$PROJECT_ID/flutter:main
```

---

`stable` is 3.19 now but I'll change this to `main` for now.